src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/latest.sql

```diff
@@ -100,6 +100,8 @@ BEGIN
     PERFORM setval('pg_catalog.pg_dist_groupid_seq', (SELECT MAX(groupid)+1 AS max_group_id FROM pg_dist_node), false);
     PERFORM setval('pg_catalog.pg_dist_node_nodeid_seq', (SELECT MAX(nodeid)+1 AS max_node_id FROM pg_dist_node), false);
     PERFORM setval('pg_catalog.pg_dist_colocationid_seq', (SELECT MAX(colocationid)+1 AS max_colocation_id FROM pg_dist_colocation), false);
+    PERFORM setval('pg_catalog.pg_dist_operationid_seq', (SELECT MAX(operation_id)+1 AS max_operation_id FROM pg_dist_cleanup), false);
+    PERFORM setval('pg_catalog.pg_dist_cleanup_recordid_seq', (SELECT MAX(record_id)+1 AS max_record_id FROM pg_dist_cleanup), false);
 
```

Comment:
we probably also need to copy over the pg_dist_cleanup table, since pg_upgrade does not relieve us from our cleanup duties

Reply:
Thanks for pointing this out. Fixed.

Reply:
@marcocitus 
The way I see it being done is that if we introduce a catalog in version X, the X+1 will need to worry about copying over the data. Since we are at version X we don't need it ?

Example : pg_dist_node_metadata was introduced in 8.0.1. Starting from 9.0.1 we start to copy data over.

Reply:
With the same logic, we don't need the two line change above as well so I have removed it.

Reply:
This is for PostgreSQL upgrades. The pg_catalog contents are lost during the upgrade (which effectively rebuilds the database from scratch), so we copy them over to the public schema and back after the upgrade.

We can do a PostgreSQL 14->15 upgrade on Citus 11.1, so we need to copy the cleanup records and sequences, to resume cleanup after the upgrade if needed.

Reply:
Got it. I have made the fix. The scripts are being called by Marlin it seems so now it makes more sense.